src/main/kotlin/com/openlattice/linking/BackgroundLinkingService.kt

```diff
@@ -59,8 +50,7 @@ internal const val MINIMUM_SCORE = 0.75
 class BackgroundLinkingService
 (
         private val executor: ListeningExecutorService,
-        private val hazelcastInstance: HazelcastInstance,
-        private val pgEdmManager: PostgresEdmManager,
+        hazelcastInstance: HazelcastInstance,
```

Comment:
why is this not private anymore?

Reply:
It's not a class field anymore, because it is only used for initialization